src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -191,19 +165,19 @@ limitations under the License.
       </mandatoryConform>
     </attribute>
     <!-- Conformance feature WDSCH - for now optional -->
-    <attribute side="server" code="20" name="NumberOfWeekDaySchedulesSupportedPerUser" define="NUM_WEEKDAY_SCHEDULES_SUPPORTED_PER_USER" type="int8u" default="0" optional="true">
+    <attribute side="server" code="20" name="NumberOfWeekDaySchedulesSupportedPerUser" define="NUM_WEEKDAY_SCHEDULES_SUPPORTED_PER_USER" type="int8u" default="0" optional="true" max="0xFD">
       <mandatoryConform>
         <feature name="WDSCH"/>
       </mandatoryConform>
     </attribute>
     <!-- Conformance feature YDSCH - for now optional -->
-    <attribute side="server" code="21" name="NumberOfYearDaySchedulesSupportedPerUser" define="NUM_YEARDAY_SCHEDULES_SUPPORTED_PER_USER" type="int8u" default="0" optional="true">
+    <attribute side="server" code="21" name="NumberOfYearDaySchedulesSupportedPerUser" define="NUM_YEARDAY_SCHEDULES_SUPPORTED_PER_USER" type="int8u" default="0" optional="true" max="0xFD">
       <mandatoryConform>
         <feature name="YDSCH"/>
       </mandatoryConform>
     </attribute>
     <!-- Conformance feature HDSCH - for now optional -->
-    <attribute side="server" code="22" name="NumberOfHolidaySchedulesSupported" define="NUM_HOLIDAY_SCHEDULES_SUPPORTED" type="int8u" default="0" optional="true">
+    <attribute side="server" code="22" name="NumberOfHolidaySchedulesSupported" define="NUM_HOLIDAY_SCHEDULES_SUPPORTED" type="int8u" default="0" optional="true" max="0xFD">
```

Comment:
Matches the matter xml - https://github.com/project-chip/connectedhomeip/blob/master/data_model/master/clusters/DoorLock.xml#L798-L817